extensions/xcode/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "xcode",
+  "title": "Xcode",
+  "version": "1.0.0",
+  "author": "Sven Tiigi",
+  "license": "MIT",
+  "description": "Open your recent Xcode Projects, view the latest Xcode Releases and manage your Xcode Simulators",
+  "icon": "icon.png",
+  "commands": [
+    {
+      "name": "projects.command",
+      "title": "Projects",
+      "subtitle": "Your recent Xcode Projects",
+      "description": "Search and open your recent Xcode Projects",
+      "mode": "view"
+    },
+    {
+      "name": "simulator-applications.command",
+      "title": "Apps",
+      "subtitle": "Apps installed on Xcode Simulators",
+      "description": "View the bundle and sandbox path of your Apps that are installed on a Xcode Simulator",
+      "mode": "view"
+    },
+    {
+      "name": "simulators.command",
+      "title": "Simulators",
+      "subtitle": "Manage your Xcode Simulators",
+      "description": "Boot and shutdown Xcode Simulators",
+      "mode": "view"
+    },
+    {
+      "name": "releases.command",
+      "title": "Releases",
+      "subtitle": "The latest Xcode Releases",
```

Comment:
Here you can either keep "Xcode" for consistency or just leave it empty since Xcode is mentioned in the title.